src/app/app.component.css

```diff
@@ -309,7 +323,7 @@ nav{
     height: 1775px;
     border: 2px #4A437F dashed;
```

Comment:
all of them have this same border, do they all share a class as well that this border could live in?